client/src/main/scala/caliban/client/Zippable.scala

```diff
@@ -0,0 +1,401 @@
+package caliban.client
+
+trait Zippable[-A, -B] {
```

Comment:
I think we need the source attribution for ZIO.

Reply:
Yep, added!